### DISCUSSION TITLE
Update failed on RPi

### DISCUSSION THREAD

**Discussion by Gerynant24** [dgreif/ring#1023] [General]
Hi,
The latest update just killed my Homebridge Pi!
It returned an error during the install and now I can't connect to my Pi homebridge at all, am getting 500 internal server error.

A restart seemed to revive the Pi but I'm wondering if it's safe to try it again.

Has anyone else attempted this update?

**Comment by dgreif** [comment]
Can you verify that you have node 16 installed? The plugin won't install properly on older versions

**Comment by Gerynant24** [comment]
Hi, yes I have v16.16.0 installed.



 

Cheers,

**Reply by dgreif** [reply]
In that case, I don't have any specific advice. Install issues are usually specific to your device/setup and aren't generally caused by the plugin itself. You can check out the [Homebridge wiki](https://github.com/homebridge/homebridge/wiki/Basic-Troubleshooting) for some troubleshooting ideas, or ask in the Homebridge discord which has "install help" channels for each platform
